src/Microsoft.VisualStudio.Editors/GlobalSuppressions.vb

```diff
@@ -421,7 +421,7 @@
 <Assembly: CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.ResourceEditor.Resource.#ctor(Microsoft.VisualStudio.Editors.ResourceEditor.ResourceFile,System.String,System.String,System.String,System.String,Microsoft.VisualStudio.Editors.ResourceEditor.Resource.ITypeResolutionContextProvider)")>
 <Assembly: CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.ResourceEditor.Resource.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)")>
 <Assembly: CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.ResourceEditor.Resource.AddFileWatcherEntry(Microsoft.VisualStudio.Editors.ResourceEditor.FileWatcher)")>
-<Assembly: CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.ResourceEditor.Resource.AdjustSystemStringAssemblyQualifiedName(System.String)~System.String")>
+<Assembly: CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Scope:="member", Target:="~M:Microsoft.VisualStudio.Editors.ResourceEditor.Resource.AdjustAssemblyQualifiedName(System.String)~System.String")>
```

Comment:
Is this because the argument `AssemblyQualifiedName` starts with a capital letter? If so, consider renaming to `assemblyQualifiedName` and removing this suppression.

Reply:
Current naming is consistent with the rest of the file. Is it worth changing this one function's style to remove the suppression?

Reply:
No, probably not.